pkg/backup/s3/s3.go

```diff
@@ -40,7 +40,7 @@ func New() (*S3, error) {
 		return nil, fmt.Errorf("%s bucket must be set", env.AWSS3Bucket)
 	}
 
-	client := s3.New(session.New())
+	client := s3.New(session.New(), &aws.Config{Region: aws.String(env.AWSS3Region)})
```

Comment:
Is this true? I found a region env here: http://docs.aws.amazon.com/sdk-for-go/api/aws/session/#env

```
AWS_REGION=us-east-1

# AWS_DEFAULT_REGION is only read if AWS_SDK_LOAD_CONFIG is also set,
# and AWS_REGION is not also set.
AWS_DEFAULT_REGION=us-east-1
```